src/lib/y2storage/setup_checker.rb

```diff
@@ -153,6 +165,31 @@ def nfs?(volume)
       end
     end
 
+    # @see #mount_warnings
+    #
+    # @param mount_point [MountPoint]
+    # @return [SetupError, nil]
+    def mount_warning(mount_point)
+      missing = mount_point.missing_mount_options
+      return if missing.empty?
+
+      # TRANSLATORS: do not translate %{opt} or %{path}. %{opt} is replaced by the name of a mount
+      # option in the singular sentence or a list of options separated by commas in the plural one.
+      # ${path} is replaced by the mount point.
+      msg = n_(
+        format(
+          "The fstab option %{opt} may be needed to properly mount %{path}.",
```

Comment:
NP: Don't know why, but for me "The _netdev fstab option" _sounds_ better.

Reply:
I don't have a strong preference, but I felt this way it would be more readable at first sight for certain option names.